packages/core/src/models/documentOptions.ts

```diff
@@ -0,0 +1,14 @@
+export enum DocumentSpec {
+  oas3 = 'oas3',
+}
+
+export enum DocumentServerType {
+  redoc = 'redoc',
+}
+
+export interface IDocumentOptions {
+  /** Target specification of our APIs, e.g. OpenAPI, Tinyspec ...etc. */
+  specs?: (string | DocumentSpec)[];
+  folderPath?: string;
+  server?: (string | DocumentServerType)[];
```

Comment:
Maybe we could rename the propety `server` and `DocumentServerType` e.g: from `server` To the `renderType`, and the `DocumentRenderType` or `DocumentRouterType` `DocumentType`?

Because seems that each document server extension still uses the same port in vulcan server with a different router.

If we use the `server`, it looks like each document server type use a different port to launch.

Reply:
Similar suggestion for `documentServerMiddleware` and `documentServer` extension, we could change the `server` to another word, e.g: render, router... and so on, to prevent each document server will create a new server with a different port

Reply:
Thanks, I think router will be better, I'll update all of them.